### PR TITLE
Bootstrap V5 Badges class

### PR DESCRIPTION
**Pull request by fchagasjr**
### What changed, and why?
Updated "badge" class declaration on elements style to match Bootstrap V5.
Previous class implementation was not supported and it was not properly being shown on page rendering. 

### Screenshots please :)
Before:
Screen Shot 2023-12-19 at 9 11 00 PM

After:
Screen Shot 2023-12-19 at 9 07 48 PM

### CODE REVIEW

**Comment by schoork** [comment]
Thank you @fchagasjr. Before I merge, have you used [View Components](https://viewcomponent.org/) before? This might be a good candidate for one.

**Comment by fchagasjr** [comment]
@schoork I've never used, but I agree it would be a good candidate. Please, don't merge it yet, I'll do the changes first.

File: app/components/badge_component.rb
Comment (schoork) [inline app/components/badge_component.rb:15]: Beautiful!

File: app/components/badge_component.rb
Comment (schoork) [inline app/components/badge_component.rb:4]: You don't technically need this line if you just use `@` notation in the style method.

File: app/helpers/sidebar_helper.rb
Comment (schoork) [inline app/helpers/sidebar_helper.rb:11]: Not using here?

File: spec/components/badge_component_spec.rb
Comment (schoork) [inline spec/components/badge_component_spec.rb:1]: Tests! Yes!

File: app/helpers/sidebar_helper.rb
Comment (fchagasjr) [reply app/helpers/sidebar_helper.rb:11]: I checked all the other instances on the browser to confirm they still kept the same design characteristics, but this function is not being called anywhere on the app to confirm this one.

File: app/components/badge_component.rb
Comment (fchagasjr) [reply app/components/badge_component.rb:4]: You're right. I guess only the `text` attribute needs to be accessible, the other ones can be private.

File: app/helpers/sidebar_helper.rb
Comment (schoork) [reply app/helpers/sidebar_helper.rb:11]: Okay. I didn't notice this wasn't an html template either. All good.

**Review by schoork — APPROVED** [review]
Great work @fchagasjr. Really impressed how well you did with the View Components on your first shot!

**Comment by fchagasjr** [comment]
@schoork thanks for the feedback! I'm glad to know that

**Pull request merged**
